packages/@webex/plugin-meetings/src/reconnection-manager/index.ts

```diff
@@ -485,14 +467,14 @@ export default class ReconnectionManager {
       const media = await this.reconnectMedia();
 
       LoggerProxy.logger.log(
-        'ReconnectionManager:index#executeReconnection --> Media reestablished'
+        'ReconnectionManager:index#executeReconnection --> webRTC media connection renewed and local sdp offer sent'
       );
       this.status = RECONNECTION.STATE.COMPLETE;
```

Comment:
This status change isn't strictly accurate either. I'm wondering if we should add a getter/setter for the status in this class and change it to complete in meeting/index.ts where we are sending ```client.media.recovered```

Reply:
I've made this change as I think it makes sense that the status of the reconnection manager is only set to complete once we have actually reestablished media (got answer and done dtls handshake) and not just all the code in reconnectionManager has ran